packages/vscode-dependencies-validation/src/autofix/depsIssues.ts

```diff
@@ -0,0 +1,59 @@
+import type { Uri } from "vscode";
+import {
+  VscodeFileEventConfig,
+  VscodeUriFile,
+  VscodeWorkspace,
+} from "../vscodeTypes";
+import {
+  ENABLE_AUTOFIX,
+  getAutoFixDelay,
+  isAutoFixEnabled,
+} from "./configuration";
+import { addPackageJsonFileWatcher as addPackageJsonFileWatcher } from "./packageJsonFileWatcher";
+import { addUnsupportedFilesWatcher } from "./unsupportedFilesWatcher";
+import { findAndFixDepsIssues } from "../util";
+import { clearDiagnostics } from "../util";
+
+export function activateDepsIssuesAutoFix(
+  vscodeConfig: VscodeFileEventConfig & VscodeUriFile
```

Comment:
are you  using the DI in method params because you find it more convinient?
- e.g: for tests?

I remember you mentioned `proxyQuire` replaced must of the need for this DI...